### PR TITLE
Replace handleServiceAgents

### PR DESCRIPTION
**Pull request by garrettqmartin8**
## Description

Refactor the tsp components to use the entities action to update service agents. Delete the code needed to support handleServiceAgents from the tsp ducks file.

## Setup

- Log into the TSP app
- Click on the "Accepted" queue
- Select a shipment
- Try to add service agents to the shipment

## Code Review Verification Steps

* [x] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/165150118) for this change

### CODE REVIEW

File: src/shared/Entities/modules/serviceAgents.js [hunk 1]
```diff
@@ -27,6 +32,30 @@ export function updateServiceAgentsForShipment(shipmentId, serviceAgents, label
   };
 }
 
+export function handleServiceAgents(shipmentId, serviceAgents) {
```
Comment (sarboc) [inline src/shared/Entities/modules/serviceAgents.js]: This looks an awful lot like `updateServiceAgentsForShipment` on line 27 of this file. Do we need both functions?

File: src/shared/Entities/modules/serviceAgents.js [hunk 2]
```diff
@@ -27,6 +32,30 @@ export function updateServiceAgentsForShipment(shipmentId, serviceAgents, label
...
+  return async function(dispatch) {
+    for (const serviceAgent in serviceAgents) {
+      /* eslint-disable security/detect-object-injection */
+      dispatch(createOrUpdateServiceAgent(shipmentId, serviceAgents[serviceAgent]));
+      /* eslint-enable security/detect-object-injection */
+    }
+  };
+}
+
+export function createOrUpdateServiceAgent(shipmentId, serviceAgent) {
```
Comment (sarboc) [inline src/shared/Entities/modules/serviceAgents.js]: With this function, we now have `createServiceAgentForShipment`, `updateServiceAgentForShipment`, and `createOrUpdateServiceAgentForShipment`. Any chance we could refactor so we use the create and update when appropriate, or just default to the `createOrUpdate` version?

Comment (garrettqmartin8) [reply src/shared/Entities/modules/serviceAgents.js]: Good call. I've pushed an update for this. Let me know what you think.

File: src/shared/Entities/modules/serviceAgents.js (hunk 1 above)
Comment (garrettqmartin8) [reply src/shared/Entities/modules/serviceAgents.js]: I've replaced `updateServiceAgentsForShipment` with `handleServiceAgents`.

**Pull request merged**